core/includes/form.inc

```diff
@@ -4470,6 +4470,17 @@ function theme_file($variables) {
   return '<input' . backdrop_attributes($element['#attributes']) . ' />';
 }
 
+ /**
+ * Processes a file upload element, make use of #multiple if present.
+ */
+function form_process_file($element) {
+  if ($element['#multiple']) {
+    $element['#attributes'] = array('multiple' => 'multiple');
```

Comment:
I would prefer this here:
```suggestion
     $element['#attributes']['multiple'] = 'multiple';
```

Reply:
The PR line is adding a new key and value. Your suggestion assumes the key already exists and is adding just a value.